nixos/modules/services/x11/xserver.nix

```diff
@@ -481,11 +483,11 @@ in
         description = "Default colour depth.";
       };
 
-      useXFS = mkOption {
-        # FIXME: what's the type of this option?
-        default = false;
+      fontPath = mkOption {
+        type = types.nullOr types.str;
+        default = null;
         example = "unix/:7100";
-        description = "Determines how to connect to the X Font Server.";
+        description = "Set the X server FontPath.";
```

Comment:
The meaning of `null` should be documented.

Reply:
changed it to reference the man page of `xorg.conf`